### PR TITLE
Fix adapter test

### PR DESCRIPTION
**Pull request by cehbrecht**
This PR fixes the adapter test for latest pytest>=5.0.0

### CODE REVIEW

**Comment by cehbrecht** [comment]
@fmigneault just an update for pytest 5.0.0

File: tests/test_adapter.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def test_adapter_factory_TestAdapter_invalid_raised():
         get_adapter_factory(settings)
         pytest.fail(msg="Invalid adapter not inheriting from 'AdapterInterface' should raise on import.")
     adapter_str = '{}.{}'.format(AdapterInterface.__module__, AdapterInterface.__name__)
-    assert adapter_str in str(err), "Expected to have full adapter import string in error message."
+    assert adapter_str in str(err.value), "Expected to have full adapter import string in error message."
```
Comment (fmigneault) [inline tests/test_adapter.py]: @cehbrecht 
I'm curious about what this changes?
I though calling `str()` would print the `TypeError(<some-error>)` representation.

Comment (cehbrecht) [reply tests/test_adapter.py]: Say have changed the `ExceptionInfo`, see release notes:
https://docs.pytest.org/en/latest/changelog.html

Comment (fmigneault) [reply tests/test_adapter.py]: Ok, I think doesn't cause much concern for this specific test.

**Pull request merged**